src/parseexcel.py

```diff
@@ -0,0 +1,54 @@
, 'ECOE51', 'ECPC70', 'ECPC71', 'ECPC72', 'ECPE70', 'ECPE71', 'ECPE72', 'ECPE73', 'ECPE74', 'ECPE75', 'ECPE76', 'ECPE', 'ECPE78', 'ECOE70', 'ECOE71', 'ECOE72', 'ECPC80', 'ECPE80', 'ECPE81', 'ECPE82', 'ECPE83', 'ECPE84', 'ECPE85', 'ECPE86', 'ECPE87', 'ECPE88', 'ECOE80', 'ECOE81', 'ECLR30', 'ECLR31', 'ECLR32', 'ECLR33', 'ECLR40', 'ECLR41', 'ECLR42', 'ECLR43', 'ECLR50', 'ECLR51', 'ECLR52', 'ECLR70', 'ECLR71', 'EEIR', 'EEPC', 'EEPC12', 'EEPC21', 'EEPC22', 'EEPC22A','EEPC24','EEPE24C','EEPC24B', 'EEPC25', 'EEPC26', 'EEPC27', 'EEPC28', 'EEPC29', 'EEPC31', 'EEPC33', 'EEPC35', 'EEPC40', 'EEPC41', 'EEPC43', 'EEPC45', 'EEPE21A', 'EEPE21B', 'EEPE21C', 'EEPE22A', 'EEPE22B', 'EEPE22C', 'EEPE31A', 'EEPE31B', 'EEPE31C', 'EEPE33A', 'EEPE33B', 'EEPE33C', 'EEPE41A', 'EEPE41B', 'EEEPE41C', 'EEPE43A', 'EEPE43B', 'EEPE43C', 'EEPE43D', 'EEPE40A', 'EEPE40B', 'EEPE40C', 'EEPE40D', 'EEPE42A', 'EEPE42B', 'EEPE42C', 'EEPE42D', 'EEPE44A', 'EEPE44B', 'EEPE44C', 'EELR12', 'EELR', 'EELR21', 'EELR25', 'EELR22', 'EELR24', 'EELR26', 'EELR28', 'EELR31', 'EELR33', 'EELR35', 'EELR43', 'EELR41', 'ITIR13', 'ITPC10', 'ITPC12', 'ITPC14', 'ITIR12', 'ITPC21', 'ITPC23', 'ITPC25', 'ITPC27', 'ITPC29', 'ITPC20', 'ITPC22', 'ITPC24', 'ITPC26', 'ITPE20', 'ITPE22', 'ITPE24', 'ITPE26', 'ITPE60', 'ITPE62', 'ITPE64', 'ITPC31', 'ITPC33', 'ITPC35', 'ITPC37', 'ITPE31', 'ITPE33', 'ITPE35', 'ITPE37', 'ITPE39', 'ITPE71', 'ITPE73', 'ITPE75', 'ITPE77', 'ITPE79', 'ITPC41', 'ITPC43', 'ITPC45', 'ITPE41', 'ITPE43', 'ITPE45', 'ITPE47', 'ITPC40', 'ITPE42', 'ITPE44', 'ITPE46', 'ITPE48', 'ITOE41', 'ITOE40', 'ITOE42', 'MEIR12', 'MEIR11', 'MEPC10', 'MEPC11', 'MEPC12', 'MELR10', 'MEPC13', 'MEPC14', 'MEPC15', 'MEPC16', 'MEPC17', 'MELR11', 'MELR12', 'MELR13', 'MEPC18', 'MEPC19', 'MEPC20', 'MEPC', 'MELR', 'MELR15', 'MELR16', 'MELR17', 'MEPE10', 'MEPE11', 'MEPE12', 'MEPE13', 'MEPE14', 'MEPE15', 'MEPC22', 'MEPC23', 'MEPC24', 'MEPC25', 'MELR18', 'MELR19', 'MELR20', 'MEPE16', 'MEPE17', 'MEPE18', 'MEPE19', 'MEPE20', 'MEPE21', 'MEPC26', 'MEPC27', 'MEPC28', 'MELR21', 'MEPE22', 'MEPE23', 'MEPE25', 'MEPE26', 'MEPE27', 'MEPE', 'MEPC29', 'MEPE29', 'MEPE30', 'MEPE31', 'MEPE32', 'MEPE33', 'MEPE37', 'MEPE39', 'MEPE41', 'MELR14', 'PHIR11', 'PHIR12A', 'PHIR12B', 'PHIR12C', 'PHIR12', 'PHOE10', 'PHOE11', 'PHOE', 'PHOE13', 'PHOE14', 'PHOE15', 'PHOE16', 'PRIR11', 'PRPC10', 'PRPC11', 'PRPC12', 'PRPC13', 'PRPC14', 'PRPC15', 'PRPC16', 'PRPC17', 'PRPC19', 'PRPL10', 'PRPL11', 'PRPL12', 'PRPL13', 'PRPC18', 'PRPC20', 'PRPC21', 'PRPL14', 'PRPL15', 'PRPL16', 'PRPE10', 'PRPE11', 'PRPE12', 'PRPE13', 'PRPE14', 'PRPE15', 'PRPC22', 'PRPC23', 'PRPC24', 'PRPC25', 'PRPL17', 'PRPL18', 'PRPE16', 'PRPE17', 'PRPE18', 'PRPE19', 'PRPE20', 'PRPE21', 'PRPC26', 'PRPC27', 'PRPC28', 'PRPL', 'PRPL20', 'PRPE23', 'PRPE24', 'PRPE', 'PRPE27', 'PRPE28', 'PRPE29', 'PRPE30', 'PRPC29', 'PRPE33', 'PRPE34', 'PRPE35', 'PRPE37', 'PRPE39', 'PRPE40', 'PRPE41', 'PRPE43', 'PRPE45', 'HSIR11', 'HSIR12', 'PHIR14', 'PHIR13', 'MAIR12','MAIR11', 'MAIR22','MAIR24','CEPC23','MAIR21','ECIR11']
```

Comment:
put this in a separate file. spare people with line wrapping on!

Reply:
Got it, latest commit has all these changes now